openwrt/configs/common.config

```diff
@@ -18,89 +18,53 @@ CONFIG_LUA_ECO_OPENSSL=y
 # CONFIG_LUA_ECO_WOLFSSL is not set
 CONFIG_OPENSSH_LIBFIDO2=y
 CONFIG_OPENSSL_ENGINE=y
+CONFIG_OPENSSL_OPTIMIZE_SPEED=y
 CONFIG_OPENSSL_PREFER_CHACHA_OVER_GCM=y
 CONFIG_OPENSSL_WITH_ASM=y
 CONFIG_OPENSSL_WITH_CHACHA_POLY1305=y
 CONFIG_OPENSSL_WITH_CMS=y
 CONFIG_OPENSSL_WITH_DEPRECATED=y
 CONFIG_OPENSSL_WITH_ERROR_MESSAGES=y
-CONFIG_OPENSSL_WITH_NPN=y
 CONFIG_OPENSSL_WITH_PSK=y
 CONFIG_OPENSSL_WITH_SRP=y
+CONFIG_OPENSSL_WITH_SSE2=y
 CONFIG_OPENSSL_WITH_TLS13=y
 CONFIG_PACKAGE_apinger=y
 CONFIG_PACKAGE_bash=y
 # CONFIG_PACKAGE_dnsmasq is not set
 # CONFIG_PACKAGE_dropbear is not set
 # CONFIG_PACKAGE_firewall4 is not set
 CONFIG_PACKAGE_iperf3=y
+# CONFIG_PACKAGE_jansson is not set
 CONFIG_PACKAGE_jq=y
-CONFIG_PACKAGE_kmod-ipt-core=y
-CONFIG_PACKAGE_kmod-nf-ipt=y
-CONFIG_PACKAGE_kmod-usb-serial=y
-CONFIG_PACKAGE_kmod-usb-serial-ftdi=y
-CONFIG_PACKAGE_kmod-usb-serial-pl2303=y
-CONFIG_PACKAGE_libbz2=y
+# CONFIG_PACKAGE_kmod-nft-fib is not set
 CONFIG_PACKAGE_libcap=y
 CONFIG_PACKAGE_libcbor=y
 CONFIG_PACKAGE_libestr=y
 CONFIG_PACKAGE_libevdev=y
 CONFIG_PACKAGE_libevent2=y
 CONFIG_PACKAGE_libfastjson=y
-CONFIG_PACKAGE_libffi=y
 CONFIG_PACKAGE_libfido2=y
 CONFIG_PACKAGE_libiperf3=y
-CONFIG_PACKAGE_libiwinfo-lua=y
-CONFIG_PACKAGE_liblua=y
+# CONFIG_PACKAGE_libmnl is not set
 CONFIG_PACKAGE_libncurses=y
-CONFIG_PACKAGE_libnetsnmp=y
+# CONFIG_PACKAGE_libnftnl is not set
 CONFIG_PACKAGE_libopenssl=y
 CONFIG_PACKAGE_libpcap=y
-CONFIG_PACKAGE_libpci=y
-CONFIG_PACKAGE_libpcre=y
 CONFIG_PACKAGE_libpopt=y
-CONFIG_PACKAGE_libpython3=y
 CONFIG_PACKAGE_libreadline=y
-CONFIG_PACKAGE_librrd1=y
-CONFIG_PACKAGE_librt=y
-CONFIG_PACKAGE_libubus-lua=y
-CONFIG_PACKAGE_libuci-lua=y
+# CONFIG_PACKAGE_libucode is not set
 CONFIG_PACKAGE_libudev-zero=y
-CONFIG_PACKAGE_libuuid=y
 CONFIG_PACKAGE_lldpd=y
-# CONFIG_PACKAGE_logd is not set
 CONFIG_PACKAGE_logrotate=y
-CONFIG_PACKAGE_lua=y
 # CONFIG_PACKAGE_nftables-json is not set
-# CONFIG_PACKAGE_odhcpd-ipv6only is not set
 CONFIG_PACKAGE_openssh-client=y
 CONFIG_PACKAGE_openssh-client-utils=y
 CONFIG_PACKAGE_openssh-keygen=y
 CONFIG_PACKAGE_openssh-server=y
-CONFIG_PACKAGE_python3-base=y
```

Comment:
removing `python3` since we dont currently rely on it for anything

Reply:
Uh, shouldn't we be going the other way... Deprecating Python2 and migrating all our dependencies to Python3?

Reply:
this only pertains to python3 on the openwrt image. I opted to remove it since we dont currently have any python scripts installed on the APs. So we wont have python3 or python2 on the APs so we save the build time and the space on the image

Reply:
good riddance!